SocialCareCaseViewerApi/V1/Controllers/SocialCareCaseViewerApiController.cs

```diff
@@ -116,6 +118,18 @@ public IActionResult GetChildrensAllocatedWorker([FromQuery] ListAllocationsRequ
             return Ok(_childrenAllocationUseCase.Execute(request));
         }
 
+        /// <summary>
+        /// Find acs allocations by Mosaic ID or officer email
+        /// </summary>
+        /// <response code="200">Success. Returns allocations related to the specified ID or officer email</response>
+        [ProducesResponseType(typeof(AscAllocationList), StatusCodes.Status200OK)]
+        [HttpGet]
+        [Route("asc_allocations")]
```

Comment:
All looks good, please just change the route to asc-allocations, instead of using underscore. I also updated the other endpoint

Reply:
I've updated the route